<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirmwarePage</class>
 <widget class="QWidget" name="FirmwarePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>348</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
        font-family: &quot;Source Sans Pro&quot;;
}

QCheckBox {
        padding: 5px;
       	background: rgb(54, 60, 70);
    	border: 1px inset rgb(18, 21, 24);
        border-radius: 3px;
}

QCheckBox:disabled {
	background: rgb(38, 43, 50);
	color: rgb(180, 180, 180);
}

QCheckBox::hover {
        border: 1px solid rgb(100, 105, 113);
}

QCheckBox:focus {
	border: 1px solid rgb(76, 157, 255);
}

QCheckBox::indicator {
    width: 15px;
    height: 15px;
    background: rgb(33, 37, 43);
}

QCheckBox::indicator:disabled {
    width: 15px;
    height: 15px;
    background: rgb(33, 37, 43);
}

QCheckBox::indicator:checked {
        image: url(&quot;://check_FILL0_wght400_GRAD0_opsz20.svg&quot;);
}

QPushButton {
	background: rgb(54, 60, 70);
    border: 1px inset rgb(18, 21, 24);
	border-radius: 3px;
	padding: 6px;
}

QPushButton:disabled {
   background: rgb(38, 43, 50);
}

QPushButton:hover {
    border: 1px solid rgb(100, 105, 113);
}

QPushButton:focus {
	border: 1px solid rgb(76, 157, 255);
}

QComboBox {
	color: white;
    background: rgb(54, 60, 70);
    border: 1px inset rgb(18, 21, 24);
	border-radius: 3px;
	padding: 5px;
}

QComboBox::disabled {
	background: rgb(38, 43, 50);
	color: rgb(180, 180, 180);
}

QComboBox:hover {
    border: 1px solid rgb(100, 105, 113);
}

QComboBox:focus {
	border: 1px solid rgb(76, 157, 255);
}

QComboBox::drop-down {
	border: none;
}

QComboBox::down-arrow {
        image: url(&quot;:/arrow_drop_down_FILL0_wght400_GRAD0_opsz20_white.svg&quot;);
 	width: 6px;
	margin-right: 4px;
	margin-bottom: 2px;
}

QLineEdit {
	color: white;
    background: rgb(54, 60, 70);
    border: 1px inset rgb(18, 21, 24);
	border-radius: 3px;
	padding: 5px;
}

QLineEdit:disabled {
	background: rgb(38, 43, 50);
	color: rgb(180, 180, 180);
}

QLineEdit:hover {
    border: 1px solid rgb(100, 105, 113);
}

QLineEdit:focus {
	border: 1px solid rgb(76, 157, 255);
}

QLabel {
	min-height: 30px;
}

QAbstractSpinBox {
	color: white;
    background: rgb(54, 60, 70);
    border: 1px inset rgb(18, 21, 24);
	border-radius: 3px;
	padding: 5px;
}

QAbstractSpinBox:disabled {
	background: rgb(38, 43, 50);
	color: rgb(180, 180, 180);
}

QAbstractSpinBox:hover {
    border: 1px solid rgb(100, 105, 113);
}

QAbstractSpinBox:focus {
	border: 1px solid rgb(76, 157, 255);
}

QAbstractSpinBox::up-button {
        image: url(&quot;:/arrow_drop_up_FILL0_wght400_GRAD0_opsz20_white.svg&quot;);
 	width: 6px;
	margin-right: 6px;
	margin-top: 4px;
}

QAbstractSpinBox::down-button {
        image: url(&quot;:/arrow_drop_down_FILL0_wght400_GRAD0_opsz20_white.svg&quot;);
 	width: 6px;
	margin-right: 6px;
	margin-bottom: 4px;
}

QAbstractSpinBox::up-button:disabled,
QAbstractSpinBox::up-button:off {
                image: url(&quot;:/arrow_drop_up_FILL0_wght400_GRAD0_opsz20.svg&quot;);
}

QAbstractSpinBox::down-button:disabled,
QAbstractSpinBox::down-button:off {
                image: url(&quot;:/arrow_drop_down_FILL0_wght400_GRAD0_opsz20.svg&quot;);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,1">
   <item row="6" column="2">
    <widget class="QLabel" name="label_26">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	min-height: 24px;
	color: rgb(100, 105, 113);
}
</string>
     </property>
     <property name="text">
      <string>CUSTOM_STATUS_SCREEN_IMAGE</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	min-height: 24px;
	color: rgb(100, 105, 113);
}
</string>
     </property>
     <property name="text">
      <string>CONFIGURATION_H_VERSION</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_24">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	min-height: 24px;
	color: rgb(100, 105, 113);
}
</string>
     </property>
     <property name="text">
      <string>SHOW_CUSTOM_BOOTSCREEN</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Author:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="4" colspan="2">
    <widget class="QCheckBox" name="uCustomStatusScreenImageCheckBox">
     <property name="text">
      <string>CUSTOM_STATUS_SCREEN_IMAGE</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Show Bootscreen?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="2">
    <widget class="QCheckBox" name="uShowCustomBootscreenCheckBox">
     <property name="text">
      <string>SHOW_CUSTOM_BOOTSCREEN</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Custom Version File Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Marlin Version:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_15">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Use Custom Bootscreen?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLineEdit" name="uAuthorNameEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QLineEdit" name="uCustomVersionFileEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_13">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	min-height: 24px;
	color: rgb(100, 105, 113);
}
</string>
     </property>
     <property name="text">
      <string>SHOW_BOOTSCREEN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>30</red>
           <green>30</green>
           <blue>30</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="PlaceholderText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>100</red>
           <green>105</green>
           <blue>113</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>Source Sans Pro</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	min-height: 24px;
	color: rgb(100, 105, 113);
}
</string>
     </property>
     <property name="text">
      <string>STRING_CONFIG_H_AUTHOR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	min-height: 24px;
	color: rgb(100, 105, 113);
}
</string>
     </property>
     <property name="text">
      <string>CUSTOM_VERSION_FILE</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4" colspan="2">
    <widget class="QCheckBox" name="uShowBootscreenCheckBox">
     <property name="text">
      <string>SHOW_BOOTSCREEN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_25">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Use Custom Status Screen?</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>02010200 (v2.1.2)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
